### DISCUSSION TITLE
Tails emulated VM with UTM

### DISCUSSION THREAD

**Discussion by colemickens** [utmapp/UTM#6261] [Ideas]
**Describe the issue**  

Tails is great, but it's also 🙁. I've constantly run into issues that occur at the intersection of their ISO/IMG build process and their blend of live-tools they use.

Anyway, I've probably tried 16+ different configurations to get Tails to boot in a UTM "emulated" virtual machine.

Since I got it working, I thought I'd go ahead and share it. There have been other users asking about.

Steps:
1. Download the Tails USB Stick image (ends in `.img`)
2. Use `truncate` to expand the USB image file, because they've "helpfully" added a feature that blocks boot if your USB is too small. I... want to say more but it was it is, I guess:
   ```
   truncate -s 10G tails.img 
   ```
5. Configure a UTM VM:
   * emulated
   * uefi enabled
   * graphics: (others might work): VGA
   * primary storage:
      * path: [path to the expanded img]
      * image type: disk image
      * interface: usb

Now, I only have an M2 Macbook Air to test on but I'm not even sure this result is really usable. I pretty much got it up, then gave up because of how slow it was.

**Comment by harryqt** [comment]
>  emulated

Why emulate? Can't we use Virtualize?

**Reply by zeroerrequattro** [reply]
Tails doesn't run on ARM chips, as far as I know it can't be virtualized on Apple Silicon Macs:
[https://tails.net/doc/about/requirements/index.en.html](https://tails.net/doc/about/requirements/index.en.html)
